test/concurrent/SynchronizedListTest.ooc

```diff
@@ -10,6 +10,12 @@ use base
 use concurrent
 use unit
 
+applyIndex: static Int = 0
+applyTestFunc := func (value: Int*) {
+	expect(value@, is equal to(applyIndex + 1))
+	applyIndex = applyIndex + 1
+}
```

Comment:
`applyIndex` is static so as @sebastianbaginski said, I suspect no context is generated, so no leaks :)